common/config/rush/pnpm-lock.yaml

```diff
@@ -1057,33 +1037,6 @@ packages:
     resolution: {integrity: sha512-HbE7iORnYcjLzKNf5mIQRJQDTsVxhoXHRWEZ6KWdGh4e7+F9xTloiBicavbSoVmlAYivenIVpryHanVwsQaHUw==}
     dev: false
 
-  /@azure/identity/1.2.5_debug@4.3.4:
```

Comment:
It seems that the current dependency snapshot has some testing errors. Could you fix them below commands?
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1925438&view=results

```
> rm ./common/config/rush/pnpm-lock.yaml
> rush update
```